src/server.js

```diff
@@ -215,5 +224,91 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_ADDRESS}/geocode`,
+  async (request, response) => {
+    let address = request.params[URL_PARAM_ADDRESS];
```

Comment:
You've already validated the address on the client-side, which is great. But it's generally good practice to perform server-side validation as well. It's an important redundancy since it's not guaranteed that the user didn't bypass client-side checks. It's less of an issue in this specific case, since the geocoder API will tell you its an invalid address anyway, but you can imagine a scenario where you're putting data into a database- you'll want to perform server-side validation as well.

Reply:
Done.